admin/pr_as_branch

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+set -e
+set -u
+
+
+# Usage
+
+usage() {
+  program=$(basename "$0");
+
+  if [ $# != 0 ]; then echo "$@"; echo ""; fi;
+
+  echo "${program}: usage:";
+  echo "    ${program} <pull_request_number> <trac_ticket_number> <branch_name>";
```

Comment:
It would be helpful to note that the branch name is not actually the full branch name and the ticket number doesn't need to be repeated.